lldb-eval/parser.cc

```diff
@@ -137,6 +137,21 @@ std::tuple<lldb::BasicType, bool> PickIntegerType(
   return {lldb::eBasicTypeUnsignedLongLong, true};
 }
 
+lldb::BasicType PickCharType(clang::CharLiteralParser& literal) {
```

Comment:
I think all the methods below are `const`, so this can be passed via a const-ref. Generally we avoid non-const references as much as possible, as it's very easy to accidentally mutate the parameter.

```suggestion
lldb::BasicType PickCharType(const clang::CharLiteralParser& literal) {
```